httpchecker.go

```diff
@@ -61,6 +61,10 @@ type HTTPChecker struct {
 	// used.
 	Client *http.Client `json:"-"`
 
+	// Host is the host to add to the request that is
+	// sent for the check
+	Host string `json:"host,omitempty"`
+
 	// Headers contains headers to added to the request
 	// that is sent for the check
 	Headers http.Header `json:"headers,omitempty"`
```

Comment:
@TrevinTeacutter You can just add a `Host` header using the `Headers` field, right?

Reply:
As checkup currently stands? No, because host is treated special and you can't just pass it into the Headers of a request object.